frb_codegen/src/generator/dart/ty_delegate.rs

```diff
@@ -81,6 +89,18 @@ impl TypeDartGeneratorTrait for TypeDelegateGenerator<'_> {
                 }
                 IrTypeTime::Duration => "return wire2apiDuration(_wire2api_i64(raw));".to_owned(),
             },
+            #[cfg(feature = "uuid")]
+            IrTypeDelegate::Uuid => {
+                "return UuidValue.fromByteList(_wire2api_uint_8_list(raw));".to_owned()
+            }
+            #[cfg(feature = "uuid")]
+            IrTypeDelegate::Uuids => "
+            final bytes = _wire2api_uint_8_list(raw);
+            return List<UuidValue>.generate(
+              bytes.lengthInBytes ~/ 16,
+              (int i) => UuidValue.fromByteList(Uint8List.view(bytes.buffer, i * 16, 16)),
```

Comment:
nit: I wonder will uuid always be 16 bytes

if so, can we define `const uuidSize = 16`

Reply:
resolved in 86edba18948fa2e64a68c0cad89bd96ad168f558